Taskfile.yml

```diff
@@ -0,0 +1,82 @@
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  VERSION:
+    sh: |
+        if [ -e .git/describe_ref ]; then
+          # Running in Concourse
+          cat .git/describe_ref
+        else
+          git describe --long --dirty --always
+        fi
+  LDFLAGS: -w -s -X main.currentVersion={{.VERSION}}
+  # must match the version in the Dockerfile:
+  # https://github.com/Pix4D/linux-image-build/blob/master/dockerfiles/fly_helper/Dockerfile
```

Comment:
is this comment correct?

I see we are using: `docker.ci.pix4d.com/aws-vault-builder`